charts/alfresco-connector-ms365/templates/ingress-conector-ms365.yaml

```
@@ -0,0 +1,38 @@
+{{- $serviceName := (include "alfresco-connector-ms365.fullname" .)  -}}
+{{- $servicePort := .Values.service.externalPort -}}
+apiVersion: {{ template "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ template "alfresco-connector-ms365.fullname" . }}
```

```suggestion
  name: {{ $serviceName }}
```
reuse whenever is possible, where $serviceName is defined

OR

get rid of $serviceName, probably before was not just calling a named template